akka-stream/src/main/scala/akka/stream/impl/fusing/Interpreter.scala

```diff
@@ -630,16 +639,27 @@ private[akka] class OneBoundedInterpreter(ops: Seq[Stage[_, _]],
       (pipeline(op): Any) match {
         case b: BoundaryStage ⇒
           b.context = new EntryState("boundary", op)
+          b.preStart(LifecycleContext)
```

Comment:
try catch? This should generate onFailure for the stage, not the interpreter, I think.